### PR TITLE
Committng project structure

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Committing Project Structurre

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
-# hcl-ecommerce
\ No newline at end of file
+# Sample AEM project template
```
[the remark this answers is not in the document]
Comment (rishabh-behl) [reply README.md:1]: readme should have description of projects and modules

File: core/pom.xml [hunk 2]
```diff
@@ -20,62 +20,120 @@
...
         <artifactId>hclecomm</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>0.0.1-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
     <artifactId>hclecomm.core</artifactId>
-    <packaging>bundle</packaging>
-    <name>hclecomm - Core</name>
-    <description>Core bundle for hclecomm</description>
+    <name>HCLECOMM Project-DcomponentGroupName=Hclecomm - Core</name>
```
Comment (rishabh-behl) [inline core/pom.xml:27]: @DeepaliAgrwl why this Name ??

File: core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java [hunk 3]
```diff
@@ -25,8 +25,10 @@
 
 import org.apache.sling.api.SlingHttpServletRequest;
 import org.apache.sling.engine.EngineConstants;
-import org.osgi.framework.Constants;
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.propertytypes.ServiceDescription;
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java:29]: @DeepaliAgrwl : why do we have this java class ?

File: core/src/main/java/com/hcl/ecomm/core/filters/package-info.java [hunk 4]
```diff
@@ -13,5 +13,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-// this file simply ensures that the resulting jar is not empty - see issue #30
-package com.hcl.ecomm.it.launcher;
+@Version("1.0")
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/filters/package-info.java:16]: @DeepaliAgrwl : why do we have this java file ?

File: core/src/main/java/com/hcl/ecomm/core/listeners/SimpleResourceListener.java [hunk 5]
```diff
@@ -16,8 +16,8 @@
 package com.hcl.ecomm.core.listeners;
 
 import org.apache.sling.api.SlingConstants;
-import org.osgi.framework.Constants;
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.propertytypes.ServiceDescription;
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/listeners/SimpleResourceListener.java:20]: @DeepaliAgrwl Why do we have this file ?

File: core/src/main/java/com/hcl/ecomm/core/listeners/package-info.java [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright 2017 Adobe Systems Incorporated
+ *  Copyright 2015 Adobe Systems Incorporated
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/listeners/package-info.java:2]: @DeepaliAgrwl : This merge is wrong 
ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/breadcrumb/clientlib/less/breadcrumb.less → core/src/main/java/com/hcl/ecomm/core/listeners/package-info.java

File: core/src/main/java/com/hcl/ecomm/core/models/package-info.java [hunk 7]
```diff
@@ -14,6 +14,6 @@
  *  limitations under the License.
  */
 @Version("1.0")
-package com.hcl.ecomm.core;
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/models/package-info.java:17]: @DeepaliAgrwl Why do we have this merge and file ?

core/src/main/java/com/hcl/ecomm/core/package-info.java → core/src/main/java/com/hcl/ecomm/core/models/package-info.java

File: core/src/main/java/com/hcl/ecomm/core/schedulers/package-info.java [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright 2017 Adobe Systems Incorporated
+ *  Copyright 2015 Adobe Systems Incorporated
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/schedulers/package-info.java:2]: @DeepaliAgrwl This merge looks worng.

File: core/src/main/java/com/hcl/ecomm/core/servlets/package-info.java [hunk 9]
```diff
@@ -0,0 +1,19 @@
+/*
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/servlets/package-info.java:1]: @DeepaliAgrwl : why do we have this ?

File: core/src/test/java/com/hcl/ecomm/core/filters/LoggingFilterTest.java [hunk 10]
```diff
@@ -0,0 +1,76 @@
+/*
```
Comment (rishabh-behl) [inline core/src/test/java/com/hcl/ecomm/core/filters/LoggingFilterTest.java:1]: @DeepaliAgrwl : why do we have this ?

File: core/src/test/java/com/hcl/ecomm/core/listeners/SimpleResourceListenerTest.java [hunk 11]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (rishabh-behl) [inline core/src/test/java/com/hcl/ecomm/core/listeners/SimpleResourceListenerTest.java:1]: @DeepaliAgrwl : why do we have this ?

File: core/src/test/java/com/hcl/ecomm/core/schedulers/SimpleScheduledTaskTest.java [hunk 12]
```diff
@@ -0,0 +1,61 @@
+/*
```
Comment (rishabh-behl) [inline core/src/test/java/com/hcl/ecomm/core/schedulers/SimpleScheduledTaskTest.java:1]: @DeepaliAgrwl : why doe we have this , we are not writing schedulers .

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@DeepaliAgrwl @rakeshkumar-verma  why has target folder beeen checked in ?
core/target/classes/com/hcl/ecomm/core/filters/LoggingFilter.class

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DeepaliAgrwl) [reply README.md]: updated

File: .gitignore [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+dependency-reduced-pom.xml
+buildNumber.properties
+.mvn/timing.properties
+
+### Vault ###
+.vlt
+
+### IntelliJ ###
+.idea/
+*.iml
```
[the remark this answers is not in the document]
Comment (DeepaliAgrwl) [reply .gitignore:76]: added in gitignore file

File: core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java (hunk 3 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java:29]: removed this file

File: core/src/main/java/com/hcl/ecomm/core/filters/package-info.java (hunk 4 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/filters/package-info.java:16]: removed the file

File: core/src/main/java/com/hcl/ecomm/core/listeners/SimpleResourceListener.java (hunk 5 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/listeners/SimpleResourceListener.java:20]: removed the file

File: core/src/main/java/com/hcl/ecomm/core/listeners/package-info.java (hunk 6 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/listeners/package-info.java:2]: @rishabh-behl I have removed this file. Please do let us know if this merge error still persists.

File: core/src/main/java/com/hcl/ecomm/core/models/package-info.java (hunk 7 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/models/package-info.java]: @rishabh-behl we don't have any file at this location 
 core/src/main/java/com/hcl/ecomm/core/package-info.java 

Also I have removed the file this location core/src/main/java/com/hcl/ecomm/core/models/package-info.java

So, Please check and let us know if the issue persists

File: core/src/main/java/com/hcl/ecomm/core/schedulers/package-info.java (hunk 8 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/schedulers/package-info.java]: @rishabh-behl the file at thi slocation core/src/main/java/com/hcl/ecomm/core/schedulers/package-info.java has been removed. Please check and let us know if the issue persists

File: core/src/main/java/com/hcl/ecomm/core/servlets/package-info.java (hunk 9 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/servlets/package-info.java]: removed the file

File: core/src/test/java/com/hcl/ecomm/core/filters/LoggingFilterTest.java (hunk 10 above)
Comment (DeepaliAgrwl) [reply core/src/test/java/com/hcl/ecomm/core/filters/LoggingFilterTest.java:1]: removed the file

File: core/src/test/java/com/hcl/ecomm/core/listeners/SimpleResourceListenerTest.java (hunk 11 above)
Comment (DeepaliAgrwl) [reply core/src/test/java/com/hcl/ecomm/core/listeners/SimpleResourceListenerTest.java:1]: removed the file

File: core/src/test/java/com/hcl/ecomm/core/schedulers/SimpleScheduledTaskTest.java (hunk 12 above)
Comment (DeepaliAgrwl) [reply core/src/test/java/com/hcl/ecomm/core/schedulers/SimpleScheduledTaskTest.java:1]: removed

Comment (DeepaliAgrwl) [reply core/src/test/java/com/hcl/ecomm/core/schedulers/SimpleScheduledTaskTest.java:1]: removed

File: core/src/test/java/com/hcl/ecomm/core/listeners/SimpleResourceListenerTest.java (hunk 11 above)
Comment (DeepaliAgrwl) [reply core/src/test/java/com/hcl/ecomm/core/listeners/SimpleResourceListenerTest.java:1]: removed

File: core/src/test/java/com/hcl/ecomm/core/filters/LoggingFilterTest.java (hunk 10 above)
Comment (DeepaliAgrwl) [reply core/src/test/java/com/hcl/ecomm/core/filters/LoggingFilterTest.java:1]: removed

File: core/src/main/java/com/hcl/ecomm/core/servlets/package-info.java (hunk 9 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/servlets/package-info.java]: removed

**Pull request closed without merging**